README.md

```diff
@@ -48,16 +49,16 @@ OPTIONS:
 ```
 
 ```
-$ cargo run -- -n 3 -m 32 -i rspammer_index -u http://api.hornet-1.testnet.chrysalis2.com/
+$ cargo run --release -- -n 3 -m 32 -i rspammer_index -u http://127.0.0.1:14265
```

Comment:
`--release` makes sense, but we shouldn't assume people are running a node on the same host.

spamming without running a node is actually the point of having a client spammer.